### ISSUE TITLE
Not able see the data of Pods/Deployment/Statefulset on the pre-configured grafana.

### ISSUE THREAD

**Issue by hamid2013** [coreos/prometheus-operator#1941]
**What did you do?**
I have just installed the kube-prometheus on my cluster.

**What did you expect to see?**
Pre-Configured Dashboard for the Pods/Deployment/Statefulset/etc

**What did you see instead? Under which circumstances?**
I can see node related graph, but when it comes to pods related workload not able to see anything. Its just giving NA.

**Environment**
Pre-Prod

* Prometheus Operator version:

	insert image tag or Git SHA here

* Kubernetes version information:

Client Version: version.Info{Major:"1", Minor:"7", GitVersion:"v1.7.1", GitCommit
:"1dc5c66f5dd61da08412a74221ecc79208c2165b", GitTreeState:"clean", BuildDate:"201
7-07-14T02:00:46Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.7", GitComm
it:"0c38c362511b20a098d7cd855f1314dad92c2780", GitTreeState:"clean", BuildDate:"2
018-08-20T09:56:31Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}

* Kubernetes cluster kind: 

	insert how you created your cluster: acs-engine.

* Manifests:

```
insert manifests relevant to the issue
```

* Prometheus Operator Logs:

```
insert Prometheus Operator logs relevant to the issue here
```

**Comment by brancz** [comment]
How did you install the stack?

**Comment by hamid2013** [comment]
I have used this one year before on k8s version 1.7.2. And cloned that repo that time, so that i can use it later. So the same repo i have used and not cloned the latest one as there are many changes happened.
So on that version we have hack/cluster-monitoring/deploy, so i have used that script to do that.
But in this case, we have the k8s version as 1.10.7

**Comment by brancz** [comment]
Yes there are lots of changes that are necessary as there were breaking changes in Kubernetes, so you will have to use the latest version of the stack.

**Comment by hamid2013** [comment]
Even in the new repo, the pod data is not coming through.

Checked the log for the kube-state-metrics, didn't find anything wrong.

**Comment by schizo99** [comment]
I deployed with the latest images today and I get the same errors as above.
Running kubernetes 1.11.3 in Azure AKS.

**Comment by mendrugory** [comment]
I deployed using coreos helm charts in azure aks with kubernetes 1.9.11


```
time="2018-10-26T12:57:00Z" level=info msg="Listening on 0.0.0.0:9100" source="node_exporter.go:76"
time="2018-10-26T12:57:15Z" level=error msg="ERROR: infiniband collector failed after 0.000851s: strconv.ParseUint: parsing \"N/A (no PMA)\": invalid syntax" source="collector.go:123"
time="2018-10-26T12:57:30Z" level=error msg="ERROR: infiniband collector failed after 0.025460s: strconv.ParseUint: parsing \"N/A (no PMA)\": invalid syntax" source="collector.go:123"
```

**Comment by schizo99** [comment]
Seems to be solved in the latest version of node-exporter. Tested with node-exporter v0.17.0-rc.0 and I then don't get the errors. So I guess we'll have to wait for the release and for the coreos helm charts to be updated.

**Comment by marcinma** [comment]
Happens still when installing through kube-prometheus/manifests both AKS 1.13.10 and 1.14.6
